src/remarkable-ui/components/editors/inputs/shared/InputField/InputField.module.css

```
@@ -0,0 +1,78 @@
+.input {
+  height: inherit;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  height: var(--em-textfield-height, 2.5rem);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)